kinetic/distribution.yaml

```diff
@@ -11689,6 +11689,21 @@ repositories:
       url: https://github.com/yujinrobot/yocs_msgs.git
       version: kinetic
     status: developed
+  yoctopuce_altimeter:
+    doc:
+      type: git
+      url: https://github.com/amstrudy/yoctopuce_altimeter.git
+      version: kinetic
```

Comment:
This is not a valid branch name. The only branch is `master`.

Reply:
Sorry, I'm a bit new to this. I see what you're saying about the release section, and I fixed that. I don't understand what you mean when you say "This is not a valid branch name. The only branch is master." Could you please elaborate on that? Thanks again.

Reply:
The github repository listed in the previous line has only one branch, named `master`. And the version entry should be a valid branch of the repository. So it shouldn't be `kinetic` but `master` instead

Reply:
Thanks! I ended up just adding a kinetic branch.